### PR TITLE
Update to Dashboard SDK 4

### PR DESCRIPTION
**Pull request by turnervink**
- Manually hydrate Twitter posts using the Twitter API
- Added config files for Hootsuite, Twitter, and Trello information
- Updated README with Twitter and Trello setup instructions
- Cleaned up stream and app popup
- Added more info from social posts to Trello cards added through the Dashboard

### CODE REVIEW

File: conf.d/default.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ server {
 
     location /api {
        rewrite ^/api/?(.*) /$1 break;
-       proxy_pass http://web:8000;
+       proxy_pass http://localhost:8000;
```
Comment (markcallen) [inline conf.d/default.conf]: This change isn't needed.  It should be web as defined in the docker-compose file.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.DS_Store
+app/node_modules
+app/config/turner.json
+static/js/vars.js
```
Comment (markcallen) [inline .gitignore]: This file shouldn't be removed as it is returned for the app to work, just put a placeholder value into any fields that need to be updated.

Comment (turnervink) [reply .gitignore]: I removed my info from `vars.js` and updated the README to indicate the developer has to fill in their Trello info there.

File: conf.d/default.conf (hunk 1 above)
Comment (turnervink) [reply conf.d/default.conf]: 👍

**Pull request merged**